vizro-ai/docs/pages/user-guides/advanced-options.md

```diff
@@ -0,0 +1,204 @@
+
+#### Vizro ready
+This fig object is in the standard `vizro_dark` theme, and can [be inserted into a Vizro dashboard](add-generated-chart-usecase.md).
+
+!!! example "Vizro fig object"
+
+    === "Code"
+        ```py
+        from vizro_ai import VizroAI
+        import plotly.express as px
+
+        df = px.data.gapminder()
+        vizro_ai = VizroAI()
+
+        res = vizro_ai.plot(df, "the trend of gdp over years in the US", return_elements=True)
+        fig = res.get_fig_object(data_frame=df, vizro=True)
+        fig.show()
+        ```
+
+    === "Result"
+        [![VizroAIChartVizro]][VizroAIChartVizro]
+        ```
+    [VizroAIChartVizro]: ../../assets/user_guides/VizroAIVizro.png
+
+#### Pure Plotly/Dash
+This fig object is a plain vanilla plotly figure.
+
+!!! example "Plotly fig object"
+
+    === "Code"
+        ```py
+        from vizro_ai import VizroAI
+        import plotly.express as px
+
+        df = px.data.gapminder()
+        vizro_ai = VizroAI()
+
+        res = vizro_ai.plot(df, "the trend of gdp over years in the US", return_elements=True)
+        fig = res.get_fig_object(data_frame=df, vizro=False)
+        fig.show()
+        ```
+
+    === "Result"
+        [![VizroAIChartPlotly]][VizroAIChartPlotly]
+        ```
+    [VizroAIChartPlotly]: ../../assets/user_guides/VizroAIPlotly.png
+
+#### Using different data
+You may want create the fig object with different data. Note that of course the overall schema should still be adhered to (or else the chart might fail), but for example certain rows can be filtered out. You can re-evaluate this function to produce various fig object for different data.
+
+!!! example "Different data"
+
+    === "Code"
+        ```py
+        from vizro_ai import VizroAI
+        import plotly.express as px
+
+        df = px.data.gapminder()
+        vizro_ai = VizroAI()
+
+        res = vizro_ai.plot(df, "the average of gdp for all continents as bar chart", return_elements=True)
+
+        # The produced chart could handle many continents, but we choose to filter for the US
+        df_us = df[df['country'] == 'United States']
+        fig = res.get_fig_object(chart_name="different_name", data_frame=df_us, vizro=True)
+        fig.show()
+        ```
+
+    === "Result"
+        [![VizroAINewData]][VizroAINewData]
+        ```
+    [VizroAINewData]: ../../assets/user_guides/VizroAINewData.png
+        ```
+
+#### Changing the chart name
+This option will execute the chart code with the name provided under `chart_name`. This can be important when try to avoid overwriting variables in the namespace.
```

Comment:
```suggestion
This option executes the chart code with the name provided under `chart_name`. This can be important when you want to avoid overwriting variables in the namespace.
```